comp110-worksheet-6/OxoBoard.cs

```diff
@@ -10,38 +10,87 @@ public enum Mark { None, O, X };
 
 	public class OxoBoard
 	{
-		// Constructor. Perform any necessary data initialisation here.
-		// Uncomment the optional parameters if attempting the stretch goal -- keep the default values to avoid breaking unit tests.
-		public OxoBoard(/* int width = 3, int height = 3, int inARow = 3 */)
+        public Mark[,] Board;
+
+        // Constructor. Perform any necessary data initialisation here.
+        // Uncomment the optional parameters if attempting the stretch goal -- keep the default values to avoid breaking unit tests.
+        public OxoBoard(/* int width = 3, int height = 3, int inARow = 3 */)
 		{
-			throw new NotImplementedException("TODO: implement this function and then remove this exception");
-		}
+            Board = new Mark[3, 3] { { 0, 0, 0 },
+                                     { 0, 0, 0 },
+                                     { 0, 0, 0 } };
+
+        }
 
 		// Return the contents of the specified square.
 		public Mark GetSquare(int x, int y)
 		{
-			throw new NotImplementedException("TODO: implement this function and then remove this exception");
+            return Board[x, y];
 		}
 
 		// If the specified square is currently empty, fill it with mark and return true.
 		// If the square is not empty, leave it as-is and return False.
 		public bool SetSquare(int x, int y, Mark mark)
 		{
-			throw new NotImplementedException("TODO: implement this function and then remove this exception");
-		}
+            if (Board[x, y] == 0)
```

Comment:
When using enums it's best to stick to the named values i.e. `Mark.None` rather than integer values